apps/system/js/app_install_manager.js

```diff
@@ -242,7 +243,9 @@
 
       if (manifest.size) {
         this.size.removeAttribute('data-l10n-id');
-        this.size.textContent = this.humanizeSize(manifest.size);
+        this.humanizeSize(manifest.size).then(val => {
+          this.size.textContent = val;
+        });
```

Comment:
Also, a style nit: you don't need curly braces in the arrow function if it only has one expression.